tests/TailwindBuilderTest.php

```diff
@@ -11,27 +11,36 @@
 
 use PHPUnit\Framework\TestCase;
 use Symfony\Component\Cache\Adapter\ArrayAdapter;
+use Symfony\Component\Filesystem\Exception\IOException;
 use Symfony\Component\Filesystem\Filesystem;
-use Symfony\Component\Finder\Finder;
 use Symfonycasts\TailwindBundle\TailwindBuilder;
 
 class TailwindBuilderTest extends TestCase
 {
     protected function setUp(): void
     {
         $fs = new Filesystem();
-        if (file_exists(__DIR__.'/fixtures/var/tailwind')) {
-            $fs->remove(__DIR__.'/fixtures/var/tailwind');
-        }
         $fs->mkdir(__DIR__.'/fixtures/var/tailwind');
     }
 
     protected function tearDown(): void
     {
-        $finder = new Finder();
-        $finder->in(__DIR__.'/fixtures/var/tailwind')->files();
-        foreach ($finder as $file) {
-            unlink($file->getRealPath());
+        $fs = new Filesystem();
+        $i = 0;
+        while (true) {
+            try {
+                $fs->remove(__DIR__.'/fixtures/var/tailwind');
+                break;
+            } catch (IOException $e) {
+                if ($i++ > 5) {
+                    // Sometimes "Permission denied" error happens on Windows,
+                    // add a warning about it
+                    $this->addWarning('Could not remove the temporary tailwind/ dir from the first time: '.$e->getMessage());
```

Comment:
To clarify this logic: we try 5 times (waiting a second in between) and if still failing, create the warning and stop trying?

Reply:
Does it really help? What if we skip this test for Windows env?
Another thing to try is creating a temp directory instead of using `Filesystem` (`sys_get_temp_dir()`)

Reply:
I say let's try this method and perhaps skip if we find it doesn't help.

Reply:
@kbond That was supposed to be so, but `addWarning()` seems does not work in `tearDown()` already (too late probably) and gives no effect, so I drop it completely.

So the changes are trying to remove that dir during 5 seconds with 1s delay, if still fails - it throws.

Let's give it a try, and if it still does not help - then we can skip, but instead of skipping the those tests completely we can just skip (suppress) this exception only for Windows OS. Because tests work well, the problem is only about clean up after the tests, and if it won't be cleaned up well in Windows only - probably not a big deal